pkg/kubelet/util/boottime_util_linux.go

```diff
@@ -21,17 +21,49 @@ package util
 
 import (
 	"fmt"
+	"os"
+	"strconv"
+	"strings"
 	"time"
 
 	"golang.org/x/sys/unix"
+	"k8s.io/klog/v2"
 )
 
-// GetBootTime returns the time at which the machine was started, truncated to the nearest second
+// GetBootTime returns the time at which the machine was started, truncated to the nearest second.
+// It uses /proc/uptime first, which is more accurate, and falls back to the less accurate
+// unix.Sysinfo if /proc/uptime failed.
 func GetBootTime() (time.Time, error) {
+	bootTime, err := getBootTimeWithProcUptime()
+	if err != nil {
+		klog.ErrorS(err, "Failed to get boot time from /proc/uptime. Will retry with unix.Sysinfo")
```

Comment:
How expected is this error? Given that there's a fallback, I wonder if this should be a lower verbosity.

Reply:
lowering to warning or info sounds good.  I think the error shouldn't be expected much, neither is it significant.
will do the update after the other comment on /proc/stat is settled.

Reply:
Done.